src/migration/10_fix_userdata_columns.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE [dbo].[USERDATA] ALTER COLUMN [strSkill] VARCHAR (20)
```

Comment:
By default right now this is set to size 10, any reason why we should update this to 20?

Reply:
@UTengine, I agree with @srmeier here. The actual DBAgent C++ code reads it as 10 bytes.
Don't worry, I'll update this with additional changes after rebasing this PR.

Reply:
Done here: https://github.com/ko4life-net/ko-db/pull/12/commits/d5563e3b4cd371986eaa836dff6022ac03d016e7

Reply:
There's a reason for it as it also did in official files, maybe someoone else knows I don't.

Reply:
> There's a reason for it as it also did in official files, maybe someoone else knows I don't.

10 is fine:
![image](https://github.com/ko4life-net/ko-db/assets/626562/72a9489b-1e66-4e8a-869f-fa2ce4da1a09)
![image](https://github.com/ko4life-net/ko-db/assets/626562/94573ddd-6a37-4b56-92c2-8188d91c19c5)